### ISSUE TITLE
Copy logs to clipboard without line numbers

### ISSUE THREAD

**Issue by AlekSi** [drone/drone-ui#54]
Due to line numbers.

![2016-09-20_14-26-24](https://cloud.githubusercontent.com/assets/11512/18670750/6495f574-7f52-11e6-8be4-4c4ee2ec8576.png)

**Comment by bradrydzewski** [comment]
this could be fixed with a few minor css modifications

disable text selection of row numbers:

```diff
.term-row>div:first-child {
    padding-right: 20px;
    color: rgba(0,0,0,.3);
+   -webkit-user-select: none;
+           user-select: none;
}
```

disable text selection of timestamps:

```diff
.term-row>div:last-child {
    padding-left: 20px;
    color: rgba(0,0,0,.3);
+   -webkit-user-select: none;
+           user-select: none;
}
```

I'm currently focused on the server-side code so probably won't have time to make any UI improvements for a while. I'm happy, however, to accept a PR for such improvements.

**Comment by bradrydzewski** [comment]
fixed with 9913c868839e709599f3ed659c3e0a137a2e2706
